app/src/main/java/com/osrsd/App.java

```diff
@@ -55,7 +61,10 @@ private static void init() {
                 new PrintVarbits(library),
                 new PrintSoundEffects(library),
```

Comment:
We should make this configurable, so we can run App with arguments like:
`print --varbits --items --soundeffects --objects`

Reply:
Yes that is a good idea.